brillig_bytecode/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "brillig_bytecode"
```

```suggestion
name = "brillig_vm"
```

This is a separate VM which we're embedding into the ACVM here so I think this rename is correct. Note that directory name needs to be updated as well.